src/app/core/workbox/settings-menu.component.ts

```diff
@@ -16,18 +15,18 @@ import {WorkboxService} from "./workbox.service";
     selector: "ct-settings-menu",
     styleUrls: ["./settings-menu.component.scss"],
     template: `
-        <ct-generic-dropdown-menu [ct-menu]="menu" menuAlign="left" [menuState]="openStatus"
-                                  [class.update-available]="global.platformIsOutdated">
+        <ct-generic-dropdown-menu [ct-menu]="menu" menuAlign="left"
+                                  [class.update-available]="global.platformIsOutdated" #settingsDropDown>
 
-            <button type="button" class="btn btn-unstyled">
+            <button type="button" class="btn btn-unstyled" (click)="settingsDropDown.toggleMenu()">
                 <span *ngIf="active">{{ userLabel }}</span>
                 <i class="fa fa-chevron-down fa-fw settings-icon"> </i>
             </button>
 
         </ct-generic-dropdown-menu>
 
         <ng-template #menu class="mr-1">
-            <ul class="list-unstyled">
+            <ul class="list-unstyled" (click)="settingsDropDown.hide()">
```

Comment:
What do you think about moving the hide call after the `(click)` calls for each of the `<li>`? It might happen one day that we have some kind of empty space in the `<ul>` that won't have any action but will close the dropdown if clicked.